cmd/dm-syncer/main.go

```diff
@@ -63,7 +63,13 @@ func main() {
 		log.L().Info("", zap.Stringer("dm-syncer conf", conf))
 	})
 
-	sync := syncer.NewSyncer(conf)
+	conf2, err := conf.DecryptPassword()
```

Comment:
Can we let the user use an unencrypted password just as old _Syncer_ did?

Reply:
update in 820012b